content/spin/v2/redis-outbound.md

```diff
@@ -90,25 +91,23 @@ You can find a complete Rust code example for using outbound Redis from an HTTP
 
 > [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-js-sdk/)
 
-Redis functions are available on [the `Redis` object](https://fermyon.github.io/spin-js-sdk/variables/Redis.html). The function names match the operations above, but you must pass the Redis instance address to _each_ operation as its first parameter. For example:
+Redis functions are available on [the `Redis` module](https://fermyon.github.io/spin-js-sdk/modules/Redis.html). The function names match the operations above. For example:
 
 ```javascript
-import {Redis} from "@fermyon/spin-sdk"
+import { Redis } from "@fermyon/spin-sdk"
 
-const value = Redis.get(address, key);
+let db = Redis.open("redis://localhost:6379")
+const value = db.get(key);
 ```
 
 **General Notes**
-
-* The `spinSdk` object is always available at runtime. Code checking and completion are available in TypeScript at design time if the module imports anything from the `@fermyon/spin-sdk` package.
-* Address and key parameters are strings.
-* Bytes parameters and return values are buffers (TypeScript `ArrayBuffer`).
+* key parameters are strings.
+* Bytes parameters and return values are buffers (TypeScript `Uint8Array`).
 * Lists are passed and returned as arrays.
-* If a Spin SDK function fails, it throws an [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error).
```

Comment:
You've removed the error info - is that intentional?  Is this sort of thing considered implicit in JS?  (Genuine question, I have no idea what is idiomatic in JS docs except for being half-as-- er anyway business before RANTY PLEASURE.)

Reply:
It is normal for JS to throw errors when things fail. I was not sure adding that generic information was going to provide much value but happy to add it back.

Reply:
Nah, no worries.